manage_breast_screening/assets/sass/main.scss

```diff
@@ -80,3 +82,45 @@ a[href="#"] {
     }
   }
 }
+
+.app-status-bar {
+  background-color: $color_app-dark-blue;
+  color: $color_nhsuk-white;
+  padding: nhsuk-spacing(2) 0;
+  border-bottom: 1px solid nhsuk-colour("grey-4");
+}
+
+.app-status-bar a {
+  color: $color_nhsuk-white;
+}
+.app-status-bar a:hover,
+.app-status-bar a:active {
+  color: $color_nhsuk-white;
+}
+.app-status-bar a:focus {
+  color: $color_nhsuk-black;
+}
+
+.app-status-bar__row {
+  display: flex;
+  flex-wrap: wrap;
+  gap: nhsuk-spacing(4);
+  align-items: center;
+}
+
+.app-status-bar__row + .app-status-bar__row {
+  margin-top: nhsuk-spacing(2);
+  padding-top: nhsuk-spacing(2);
+  border-top: 1px solid rgba($color_nhsuk-white, 0.2);
+}
+
+.app-status-bar__item {
+  display: flex;
+  gap: nhsuk-spacing(2);
+  align-items: center;
+}
+
+.app-status-bar__key {
+  @include nhsuk-typography-weight-bold(true);
+  opacity: 0.9;
+}
```

Comment:
I imagine it might be good to put these in their own named file inside a folder like `_components`?